docs/openapi/components/schemas/credentials/CommercialInvoiceCertificate.yml

```diff
@@ -56,6 +56,7 @@ example: |-
       "CommercialInvoiceCertificate"
     ],
     "name": "Commercial Invoice Certificate",
+    "description": "Document recording a transaction between the seller and the buyer",
```

Comment:
This works unless either seller or buyer may be plural...
```suggestion
    "description": "Document recording a transaction between a seller and a buyer",
```

Reply:
Thanks.